### ISSUE TITLE
[Feature] Axis-aligned particle rendering

### ISSUE THREAD

**Issue by dldev32** [DonBruce64/MinecraftTransportSimulator#1470]
By default, all particles are played by turning towards the player. With the help of this feature, that is, the parameter, we allow playing a particle fixed on any specified axis. This will allow correct rendering, for example, of muzzle flash particles, as well as correct rendering of the particle in cameras.

Sample:

```json
      {
        "type": "generic",
        "isAxisAligned": true, //allows AA particles
        "rot": [0.5, 1.0, 0.1], //fixed rotation params for AA particles
        "isBright": true,
        "quantity": 1,
        "duration": 16,
        "scale": 4.0,
        "transparency": 0.3,
        "textureList": []
        "textureDelays": [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
        "pos": [-0.004686, 0.268425, 1.22597],
        "initialVelocity": [0.0, 0.0, 0.0],
        "movementVelocity": [0.0, 0.0, 0.0],
        "terminalVelocity": [0.0, 0.0, 0.0],
        "activeAnimations": [
          {
            "animationType": "visibility",
            "variable": "gun_muzzleflash",
            "clampMin": 1.0,
            "clampMax": 1.0
          },
        ]
      }
```

**Comment by Lila-Bat-Huntress** [comment]
an interesting idea, but i dont know if using particles for flashes is gonna be as good as a model object

**Comment by dldev32** [comment]
> an interesting idea, but i dont know if using particles for flashes is gonna be as good as a model object

no one game has a muzzleflashes, which using a models, so with particles it will look much prettier

**Comment by Wartich** [comment]
Yeah, that would've been really good feature

**Comment by boot2big** [comment]
I personally use particles for muzzle flashes myself, maybe check them out? @Lila-Bat-Huntress Not saying it will change your opinion either way, but I like this option more as it's a separate asset from the gun model itself, and is universally applicable regardless of how much texture space is left on the gun's texture sheet.

**Comment by DonBruce64** [comment]
Model or particle, you can get the same effect either way.  This is more just a convenience thing to use the same flash for all guns and not need to re-create the model bit.  Either DL drops a pack to get this in the next month or so, or one of youse makes this a PR, or I drop it.  Y'all are out of credits for features right now.

**Comment by DonBruce64** [comment]
Model or particle, you can get the same effect either way.  This is more just a convenience thing to use the same flash for all guns and not need to re-create the model bit.  Either DL drops a pack to get this in the next month or so, or one of youse makes this a PR, or I drop it.  Y'all are out of credits for features right now.

**Comment by DonBruce64** [comment]
Implemented in V22.6.0.
